data/templates/wg21.html

```diff
@@ -62,7 +62,7 @@ <h3 class="subtitle" style="text-align:center">$subtitle$</h3>
 <table style="border:none;float:right">
   <tr>
     <td>Document #:</td>
-    <td>$document$</td>
+    <td>$if(draft)$D$else$P$endif$$document$R$revision$ <a href="https://wg21.link/P$document$">[Latest]</a> <a href="https://wg21.link/P$document$/status">[Status]</a></td>
```

Comment:
We can just keep `document` untouched and use it rather than reconstructing it. Let's conditionally add the `[Latest] [Status]` if we got the `number` field:
```suggestion
    <td>
      $document$
$if(number)$
      <a href="https://wg21.link/P$number$">[Latest]</a>
      <a href="https://wg21.link/P$number$/status">[Status]</a>
$endif$
    </td>
```